gradle/libs.versions.toml

```diff
@@ -5,12 +5,12 @@ assertJ = "3.26.3"
 # match with <root>/settings.gradle.kts
 awsSdk = "2.26.25"
 commonmark = "0.22.0"
-detekt = "1.23.6"
+detekt = "1.23.7"
 intellijExt = "1.1.8"
 # match with <root>/settings.gradle.kts
-intellijGradle = "2.0.0"
+intellijGradle = "2.1.0"
 intellijRemoteRobot = "0.11.22"
-jackson = "2.16.1"
+jackson = "2.17.2"
```

Comment:
We had reverted this to 2.16.1 in a previous version: https://github.com/aws/aws-toolkit-jetbrains/pull/4732/files because it broke chat, just confirming if we know if that's been fixed?

Reply:
yes we now remove the conflicting jackson jars from our distribution